cnc_ctrl_v1/Report.cpp

```diff
@@ -236,7 +236,20 @@ void  returnPoz(){
     static unsigned long lastRan = millis();
     
     if (millis() - lastRan > POSITIONTIMEOUT){
+        char tmpbuf[80], buflen=0;
+        memset(tmpbuf, 0, sizeof(tmpbuf));
```

Comment:
I don't know if this is ideal, creating and deleting a 160 byte array 5 times a second.

Reply:
Now... that seems silly.  It's one of those things where I thought I was being nice and encapsulated, but it does seem like a lot of bull... fixed.